packages/manager/modules/nasha/README.md

```diff
@@ -0,0 +1,2 @@
+- Checker les components bindings / resolve de chaque module
+- bug si pas de customName sur le service => mettre le nom du service
```

Comment:
Maybe translate in english.

Reply:
:+1: 
Removed here : https://github.com/ovh/manager/pull/7495/files#diff-d47fe37417d7c525bcf8b9aef8630b63e1cb822bca76a0acf023dd106114da35L1-L2